controller/tap/server.go

```diff
@@ -368,6 +410,7 @@ func (s *GRPCTapServer) translateEvent(orig *proxy.TapEvent) *public.TapEvent {
 							Scheme:    scheme(orig.RequestInit.GetScheme()),
 							Authority: orig.RequestInit.Authority,
 							Path:      orig.RequestInit.Path,
+							Headers:   headers(orig.RequestInit.GetHeaders()),
```

Comment:
Is it possible for `GetHeaders()` to return nil? If yes, we might want to add a check in the `headers()` closure.

Reply:
Good call. I can add an initial `nil` check in the closure.